inc/core/factory.php

```diff
@@ -69,6 +69,11 @@ public function load_modules() {
 	 */
 	public function build( $class ) {
 		$full_class_name = $this->namespace . $class;
+
+		if ( defined( 'NEVE_PRO_VERSION' ) && substr( $full_class_name, 0, 18 ) === '\Neve_Pro\Modules\\' && method_exists( $full_class_name, 'get_instance' ) ) {
+			return $full_class_name::get_instance();
+		}
```

Comment:
I think we can use the constant `NEVE_PRO_COMPATIBILITY_FEATURES`
that is defined here:
https://github.com/Codeinwp/neve-pro-addon/blob/37f18538240cb802ae51c9470018ff1a07efbb51/neve-pro-addon.php#L285-L299

Reply:
@preda-bogdan thank you for your review, I've fixed it. 🚀